xmtp_mls/src/subscriptions/stream_messages/test_case_builder.rs

```diff
@@ -1,364 +0,0 @@
-use std::collections::HashMap;
```

Comment:
i'm deleting these tests b/c they're complex, and have ambiguous value given we will be re-working streams with reliable streams update. since there aren't any big changes to streams other than adding an originator id, we can rely on the stateful tests until the reliable stream update